addon/components/polaris-button.js

```diff
@@ -156,7 +156,7 @@ export default Component.extend({
    *
    * @property icon
    * @public
-   * @type {SVG}
+   * @type {SVG Source|Component}
```

Comment:
This was my bad when I initially wrote this, but how do you feel about changing `SVG Source` to `String`, since that's the actual type of the `icon` property in `ember-polaris`?

Reply:
I also thought that, I figured it said SVG for a reason so I left it, but yes I will change 🕺